install/kubernetes/helm/istio/charts/mixer/templates/create-custom-objects-job.yaml

```diff
@@ -0,0 +1,35 @@
+{{- $istio_namespace := .Values.global.namespace }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+  labels:
+    istio: mixer
+  name: mixer-create-custom-objects-job
```

Comment:
[Pilot also has a whole swath of CRDs](https://github.com/istio/istio/blob/1c498e7712badd8f3e885fbe3b77e4d2d3621843/install/kubernetes/templates/istio-pilot.yaml.tmpl#L4-L45), we need to create them as well. Lets rename this "istio-create-custom-objects-job" and include them too.

Reply:
nit: And maybe "custom-resources-job" instead of "custom-objects-job"?

Reply:
is inside `mixer` subchart (folder), pilot's CRD are inside `pilot` subchart. Also, didn't find any custom object for pilot that depend on CRD, that's why it doesn't need a post-hook job like in mixer chart.

Reply:
Oh good point - yeah we don't create any Pilot resources at startup so there's no ordering requirements for Pilot CRDs.